src/win/winpty.cc

```diff
@@ -93,22 +105,24 @@ static NAN_METHOD(PtyGetProcessList) {
     return;
   }
 
-  HANDLE handle = reinterpret_cast<HANDLE>(static_cast<int64_t>(info[0]->Int32Value(Nan::GetCurrentContext()).FromJust()));
-  winpty_t *pc = get_pipe_handle(handle);
+  DWORD pid = info[0]->Uint32Value(Nan::GetCurrentContext()).FromJust();
+  winpty_t *pc = get_pipe_handle(pid);
   if (pc == nullptr) {
-    CloseHandle(handle);
     info.GetReturnValue().Set(Nan::New<v8::Array>(0));
     return;
   }
   int processList[64];
   const int processCount = 64;
   int actualCount = winpty_get_console_process_list(pc, processList, processCount, nullptr);
-
-  v8::Local<v8::Array> result = Nan::New<v8::Array>(actualCount);
-  for (int i = 0; i < actualCount; i++) {
+  if (actualCount <= 0) {
+    info.GetReturnValue().Set(Nan::New<v8::Array>(0));
+    return;
+  }
+  uint32_t actualCountSize = static_cast<uint32_t>(actualCount);
+  v8::Local<v8::Array> result = Nan::New<v8::Array>(actualCountSize);
+  for (uint32_t i = 0; i < actualCountSize; i++) {
```

Comment:
We can't use `size_t` here because we'd get a conversion warning during `Nan::Set`, but we can at least use `uint32_t` instead of just `int` here.